docs/standard/microservices-architecture/docker-application-development-process/docker-app-development-workflow.md

```diff
@@ -236,7 +236,7 @@ services:
 
 Note that this docker-compose.yml file is a simplified and merged version. It contains static configuration data for each container (like the name of the custom image), which always applies, plus configuration information that might depend on the deployment environment, like the connection string. In later sections, you will learn how you can split the docker-compose.yml configuration into multiple docker-compose files and override values depending on the environment and execution type (debug or release).
 
-The docker-compose.yml file example defines five services: the webmvc service (a web application); two microservices (ordering.api and basket.api); and two data source containers, sql.data based on SQL Server for Linux running as a container, and postgres.data with a Redis cache service. Each service will be deployed as a container, so a Docker image is required for each.
+The docker-compose.yml file example defines five services: the webmvc service (a web application); two microservices (ordering.api and basket.api); and two data source containers (sql.data based on SQL Server for Linux running as a container, and postgres.data as a Postgres database). Each service will be deployed as a container, so a Docker image is required for each.
```

Comment:
Thanks again @timotei for fixing up the topic.

I suggest that if parentheses are used here that the list become comma-separated (i.e., there is no longer a need to have colons, which existed because there were commas in the original list items). I suggest ... 

> The docker-compose.yml file example defines five services: the webmvc service (a web application), two microservices (ordering.api and basket.api), and two data source containers (sql.data based on SQL Server for Linux running as a container, and postgres.data as a Postgres database). Each service will be deployed as a container, so a Docker image is required for each.

Let's *NOT* sweat the other little nits I see ... The whole topic needs a little guidelines-adherence help, and we can't really handle it in a patch.

If you prefer that I fix the colons-to-commas, I can accept this PR and then address that later.